skl2onnx/operator_converters/GradientBoosting.py

```diff
@@ -22,10 +22,20 @@ def convert_sklearn_gradient_boosting_classifier(scope, operator, container):
 
     if op.n_classes_ == 2:
         transform = 'LOGISTIC'
-        base_values = [op.init_.prior]
+        # class_prior_ was introduced in scikit-learn 0.21.
+        if hasattr(op.init_, 'class_prior_'):
+            base_values = op.init_.class_prior_
+            assert base_values.shape == (2, )
+        else:
+            base_values = [op.init_.prior]
     else:
         transform = 'SOFTMAX'
-        base_values = op.init_.priors
+        # class_prior_ was introduced in scikit-learn 0.21.
+        if hasattr(op.init_, 'class_prior_'):
```

Comment:
This could be condensed into:
base_values = op.init_.class_prior_ if hasattr(op.init_, 'class_prior_') else op.init_.priors

Reply:
yes but the size limit for a line is 79 characters and I never know how flake8 behaves on this.

Reply:
You can always do this:
base_values = (op.init_.class_prior_ 
                          if hasattr(op.init_, 'class_prior_') else op.init_.priors)

Reply:
It somehow messed up my spacing, but what I meant to say is you can use () to split it over multiple lines. I feel it's more Pythonic than using if-else, but it's your call if you want to change it.